nohwookjin/src/components/Comment.tsx

```diff
@@ -0,0 +1,138 @@
+import { ChangeEvent, FormEvent, KeyboardEvent, useState } from "react";
+import styled from "styled-components";
+import profile from "../../public/images/abstract-user-flat-4.png";
+import Image from "next/image";
+
+export interface IComment {
+  id: number;
+  name: string;
+  comment: string;
+}
+
+const Comment = () => {
+  const [comment, setComment] = useState<IComment[]>([]);
```

Comment:
```suggestion
  const [comments, setComments] = useState<IComment[]>([]);
```
복수형의 이름이 더 어울릴 거 같아요 ~